client/src/plugins/user-journey-statistics/event-handlers/__tests__/ModelingEventHandlerSpec.js

```diff
@@ -85,25 +85,84 @@ describe('<ModelingEventHandler>', () => {
     });
 
 
-    it('shoud trasform data', () => {
+    describe('should transform data', () => {
+
+      it('diagram elements', () => {
+
+        callSubscriber({
+          on: (_, callback) => {
+            callback({
+              name: 'some.event',
+              data: {
+                nonAnElement: 'someValue',
+                element: create('shape'),
+                elementArray: [ create('shape') ]
+              }
+            });
+          }
+        });
 
-      callSubscriber({
-        on: (_, callback) => {
-          callback({
-            name: 'some.event',
-            data: {
-              nonAnElement: 'someValue',
-              element: create('shape'),
-              elementArray: [ create('shape') ]
+        expect(track).to.have.been.calledWith('some:event', {
+          nonAnElement: 'someValue',
+          element: {},
+          elementArray: [ {} ]
+        });
+      });
+
+
+      describe('popupmenu.Trigger - element templates id', () => {
```

Comment:
```suggestion
      describe('popupMenu.trigger - element templates id', () => {
```

Is that what we want to express?

Reply:
Yes, typo on my end